src/main/java/jenkins/branch/OrganizationFolder.java

```diff
@@ -263,9 +263,20 @@ public View getPrimaryView() {
         return super.getPrimaryView();
     }
 
+    /**
```

Comment:
@jglick Done https://github.com/jenkinsci/branch-api-plugin/pull/39/commits/db00b5f63da3201ac427110822ff9b23dc95cc96 I tried to avoid unnecessary changes but obviously, it makes sense the change.